pages/login.page.test.tsx

```diff
@@ -176,7 +176,10 @@ describe('Login - OKTA', () => {
         name: /find help/i,
       });
       await waitFor(() => {
-        expect(findHelpLink).toHaveAttribute('href', 'https://help.mpdx.org');
+        expect(findHelpLink).toHaveAttribute(
+          'href',
+          process.env.HELPJUICE_KNOWLEDGE_BASE_URL,
```

Comment:
Can you set `process.env.HELPJUICE_KNOWLEDGE_BASE_URL` to a value at the top of this test? That gives more confidence that the test isn't just passing because that variable is missing or a blank string.